SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -576,10 +576,20 @@ void USpatialActorChannel::OnCreateEntityResponse(const Worker_CreateEntityRespo
 
 	if (Op.status_code != WORKER_STATUS_CODE_SUCCESS)
 	{
-		UE_LOG(LogSpatialActorChannel, Error, TEXT("Failed to create entity for actor %s: %s"), *Actor->GetName(), UTF8_TO_TCHAR(Op.message));
-		Sender->SendCreateEntityRequest(this, GetPlayerWorkerId());
+		// Temporary hack to avoid failure to reserve entities due to timeout on large maps
```

Comment:
Can you create a ticket for investigating a real fix and put it in the comment?

Reply:
https://improbableio.atlassian.net/browse/UNR-630